catalog/transformer/catalog-transformer-geojson-metacard/pom.xml

```diff
@@ -46,6 +45,9 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Embed-Dependency>
+                            catalog-core-api-impl;scope=!test
```

Comment:
Just curious. Why do we now need to embed `catalog-core-api-impl` in the bundles?